src/Symfony/Component/Intl/DateFormatter/DateFormat/FullTransformer.php

```diff
@@ -69,16 +69,6 @@ public function __construct($pattern, $timezone)
         ];
     }
 
-    /**
-     * Return the array of Transformer objects.
-     *
-     * @return Transformer[] Associative array of Transformer objects (format char => Transformer)
-     */
-    public function getTransformers()
```

Comment:
This is an internal class

Reply:
I think we should change this class on 4.4
I get it's internal, but let's keep 3.4 super-unstable.

Reply:
change reverted while merging

Reply:
#33056